src/components/DataBrowser.vue

```diff
@@ -294,7 +285,7 @@ girder-data-table.girder-file-browser(
     :pagination.sync="pagination",
     :total-items="totalItems",
     :loading="loading",
-    :selection="selection_",
+    :selection="internalSelectable",
```

Comment:
> selection -> selectable because I think it was overlooked in #125

> prefer properties named internalProperty to property_ or _property because it's so hard to notice the underscore and recognize which thing is correct when debugging. I've noticed internalProperty used in Vuetify and I think it'd be good to adopt in the future.

I missed the rename on the left hand side, but the right hand side was intentional.

Reply:
Oh, yes. The right side is good. I mean the left side.

Reply:
I misread your first comment.  ty.